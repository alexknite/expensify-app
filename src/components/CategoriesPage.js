import React from 'react';
import CategoriesList from './CategoriesList';
import CategoriesSummary from './CategoriesSummary';

const CategoriesPage = () => (
  <div>
    <CategoriesSummary />
    <CategoriesList />
  </div>
);

export default CategoriesPage;
