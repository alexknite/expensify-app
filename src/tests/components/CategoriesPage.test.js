import React from 'react';
import { shallow } from 'enzyme';
import CategoriesPage from '../../components/CategoriesPage';

test('should render CategoriesPage correctly', () => {
  const wrapper = shallow(<CategoriesPage />);
  expect(wrapper).toMatchSnapshot();
});
