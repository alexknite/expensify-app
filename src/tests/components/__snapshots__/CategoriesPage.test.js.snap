// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render CategoriesPage correctly 1`] = `
<div>
  <CategoriesSummary />
  <Connect(CategoriesList) />
</div>
`;
